Gen5Dashboard.pro

```diff
@@ -150,12 +150,13 @@ HEADERS  += \
     Gen5Dashboard/Gen5Dashboard.h \
     ViewLayer/DisplayView/DisplayView.h \
     ViewLayer/I_SolarCarWindow/I_SolarCarWindow.h \
-    ViewLayer/CompactUI/CompactUI.h \
-    ViewLayer/CompactView/CompactView.h \
-    ViewLayer/CompactUI/I_CompactUI.h
+    ViewLayer/DisplayDashboardView/DisplayDashboardView.h \
+    ViewLayer/DisplayDashboardUI/DisplayDashboardUI.h \
+    ViewLayer/DisplayDashboardUI/I_DisplayDashboardUI.h
 
 FORMS    += \
-    CompactUI.ui
+    CompactUI.ui \
```

Comment:
I don't think we'll be using the CompactUI anymore, so we might as well get rid of this too.